src/thor/unidirectional_astar.cc

```diff
@@ -311,7 +307,7 @@ inline bool UnidirectionalAStar<expansion_direction, FORWARD>::ExpandInner(
                              (pred.not_thru_pruning() || !meta.edge->not_thru()),
                              (pred.closure_pruning() || !(costing_->IsClosed(meta.edge, tile))),
                              static_cast<bool>(flow_sources & kDefaultFlowMask),
-                             costing_->TurnType(meta.edge->localedgeidx(), nodeinfo, meta.edge),
+                             costing_->TurnType(pred.opp_local_idx(), nodeinfo, meta.edge),
```

Comment:
It isn't related to the main purpose of the PR but I found it while writing a unit test